classes/Currency.php

```diff
@@ -610,6 +612,27 @@ public static function getIdByIsoCode($isoCode, $idShop = 0, $forceRefreshCache
         return Cache::retrieve($cacheId);
     }
 
+    /**
+     * Get Currency ISO Code by ID
+     *
+     * @param int $id
+     * @param bool $forceRefreshCache
+     *
+     * @return string
+     */
+    public static function getIsoCodeById(int $id, bool $forceRefreshCache = false)
+    {
+        $cacheId = 'Currency::getIsoCodeById' . pSQL($id);
+        if ($forceRefreshCache || !Cache::isStored($cacheId)) {
+            $result = Currency::getCurrencyInstance($id);
```

Comment:
Could you please make a query directly to the datatase, so that we don't instanciate a Currency object (even though `getCurrencyInstance` stores the objects and there is a cache on the result There's too much magic in the object model instanciation. So instead use Db and DbQuery object, this will minimize the data fetched as well.

Reply:
I forgot it.

Reply:
Fixed